### ISSUE TITLE
IndexError: list index out of range     

### ISSUE THREAD

**Issue by hawesie** [strands-project/strands_navigation#157]
```
[INFO] [WallTime: 1416758059.995891] Preempt on state machine requested!                                                                        │···························································
[INFO] [WallTime: 1416758059.996490] Concurrent state 'magnetic_strip' returned outcome 'preempted' on termination.                             │···························································
[INFO] [WallTime: 1416758060.004407] Concurrent state 'service_pause_nav' returned outcome 'preempted' on termination.                          │···························································
[INFO] [WallTime: 1416758060.010669] Concurrent state 'pad_pause_nav' returned outcome 'preempted' on termination.                              │···························································
[INFO] [WallTime: 1416758060.015968] Concurrent state 'stuck_on_carpet' returned outcome 'preempted' on termination.                            │···························································
[INFO] [WallTime: 1416758060.017863] Concurrent state 'bumper' returned outcome 'preempted' on termination.                                     │···························································
[INFO] [WallTime: 1416758060.295990] going to waypoint in node resulted in                                                                      │···························································
False                                                                                                                                           │···························································
[INFO] [WallTime: 1416758060.413156] /topological_navigation: Navigating From WayPoint6 to WayPoint5                                            │···························································
[ERROR] [WallTime: 1416758060.413945] Exception in your execute callback: list index out of range                                               │···························································
Traceback (most recent call last):                                                                                                              │···························································
  File "/opt/ros/hydro/lib/python2.7/dist-packages/actionlib/simple_action_server.py", line 299, in executeLoop                                 │···························································
    self.execute_callback(goal)                                                                                                                 │···························································
  File "/opt/ros/hydro/lib/topological_navigation/navigation.py", line 161, in executeCallback                                                  │···························································
    self.navigate(goal.target)                                                                                                                  │···························································
  File "/opt/ros/hydro/lib/topological_navigation/navigation.py", line 224, in navigate                                                         │···························································
    route = self.search_route(Onode, target)                                                                                                    │···························································
  File "/opt/ros/hydro/lib/topological_navigation/navigation.py", line 299, in search_route                                                     │···························································
    children=to_expand[exp_index]._get_Children()                                                                                               │···························································
IndexError: list index out of range                                                                                                             │···························································
                                                                                                                                                │···························································
[INFO] [WallTime: 1416758060.723843] Waiting for services..
```

**Comment by Jailander** [comment]
@hawesie can you check the edges? maybe you lost an edge that happens when there is no possible route

**Comment by Jailander** [comment]
I will add an exception though

**Comment by hawesie** [comment]
Yes, turned out we had a one-way edge. Please handle the failure gracefully though.

**Comment by Jailander** [comment]
yes I will fix it

**Comment by Jailander** [comment]
closed via #158
